pkg/model/gcemodel/autoscalinggroup.go

```diff
@@ -128,7 +128,7 @@ func (b *AutoscalingGroupModelBuilder) buildInstanceTemplate(c *fi.ModelBuilderC
 			}
 
 			switch ig.Spec.Role {
-			case kops.InstanceGroupRoleMaster:
+			case kops.InstanceGroupRoleMaster, kops.InstanceGroupRoleAPIServer:
```

Comment:
Do APIServer nodes really need DNS permissions?

Should't we tag them with InstanceGroupRoleAPIServer?

Reply:
Great points both - will fix!

Reply:
Actually, the tag one is trickier... we share security groups on AWS as well, and this tag is a network tag, controlling the GCE equivalent of AWS security groups.

Reply:
Could use a comment then.